src/cmd.rs

```diff
@@ -47,17 +43,17 @@ pub fn execute(
         Some((version::NAME, _)) => handle(version::execute()),
         _ => {
             log::error!("unknown command");
-            process::exit(1);
+            std::process::exit(1);
```

Comment:
Is there an idiomatic reason for dropping the `use std::process;` and calling `std::process` instead?

Reply:
That makes sense. Thanks!